packages/core/ios/RNSentry.mm

```diff
@@ -382,6 +382,17 @@ - (NSDictionary*) fetchNativeStackFramesBy: (NSArray<NSNumber*>*)instructionsAdd
 
     [serializedScope setValue:contexts forKey:@"contexts"];
     [serializedScope removeObjectForKey:@"context"];
+        
+    // Remove react-native breadcrumbs
+    NSMutableArray<NSDictionary<NSString *, id> *> *breadcrumbs = [serializedScope[@"breadcrumbs"] mutableCopy];
+    for (NSInteger i = breadcrumbs.count - 1; i >= 0; i--) {
```

Comment:
This is unsafe as we are removing the items based on index while using the index to iterate over the array.

Reply:
True, the backward loop should be ok. Thank you for changing it for the predicate.